fastly/resource_fastly_tls_subscription.go

```diff
@@ -70,10 +70,34 @@ func resourceFastlyTLSSubscription() *schema.Resource {
 			},
 			"managed_dns_challenge": {
 				Type:        schema.TypeMap,
-				Description: "The details required to configure DNS to respond to ACME DNS challenge in order to verify domain ownership.",
+				Description: "(DEPRECATED) The details required to configure DNS to respond to ACME DNS challenge in order to verify domain ownership.",
```

Comment:
I believe there's `Deprecated` field on the `schema.Schema` type too, which might be worth using. I'm not sure whether that gets picked up by `terraform-plugin-docs` however, so you might still have to keep the description as you have it now

Reply:
Ah, that's good to know. Let me test that.

Reply:
https://www.terraform.io/docs/extend/best-practices/deprecations.html#renaming-a-required-attribute

> Add Deprecated to the schema definition of the existing (now the "old") attribute, noting to use the new attribute in the message.

I thought that would be helpful, but seems the `Deprecated` field doesn't really do anything for `Computed` Schema. I've tested but nothing showed up in my console, which is understandable. So I think we're okay with the current for now. cc: @Integralist

Reply:
Ah, sorry, I think I misunderstood it. It's just used for documentation, I see. I've just pushed a commit to set that field. Thanks